src/tests/file-sys-add-fixed-map-api-test/file-sys-add-fixed-map-api-test.c

```diff
@@ -0,0 +1,78 @@
+/*
+ *      Copyright (c) 2020, United States government as represented by the
+ *      administrator of the National Aeronautics Space Administration.
+ *      All rights reserved. This software was created at NASA Goddard
+ *      Space Flight Center pursuant to government contracts.
+ *
+ *      This is governed by the NASA Open Source Agreement and may be used,
+ *      distributed and modified only according to the terms of that agreement.
+ */
```

Comment:
Copyright/license will need an update merged after the license transition (doesn't look like this will get in before the transition).

Reply:
@yammajamma can you update the license based on the Apache one?